diff --git a/examples/application/application.pro b/examples/application/application.pro
index 9927010..6feb460 100644
--- a/examples/application/application.pro
+++ b/examples/application/application.pro
@@ -5,7 +5,7 @@ SOURCES += qmlog-example.cpp
 INCLUDEPATH += ../../src/ ../../
 
 QMAKE_LIBDIR_FLAGS += -L../../src
-LIBS += -lqmlog
+LIBS += -lqmlog -lrt
 
 target.path = $$(DESTDIR)/usr/bin
 
diff --git a/src/qmlog.prf b/src/qmlog.prf
index 8b6e834..ec24c52 100644
--- a/src/qmlog.prf
+++ b/src/qmlog.prf
@@ -1 +1 @@
-LIBS += -lqmlog
+LIBS += -lqmlog -lrt
